src/tests/ftest/io/ior_small.py

```diff
@@ -75,6 +75,11 @@ def test_ior_small(self):
                     # update transfer and block size
                     self.ior_cmd.transfer_size.update(test[0])
                     self.ior_cmd.block_size.update(test[1])
+                    # Skip test issue seen while advancing hdf5 vol-daos code
+                    # That is still needed due to support for new libdaos v1 API
+                    if (api == "HDF5" and test[0] == "1M" and
+                            test[1] == "32M"):
+                        self.cancelForTicket("DAOS-6427")
```

Comment:
To allow the other tests to run, I'd suggest the follwoing:
```suggestion
                        self.log.info(
                            "** SKIPPING test case: api=%s, test[0]=%s, for %s",
                            api, test[0], "DAOS-6427"
                        skip_test_at_end = True
                        continue
```

Then after line 111 add the following:
```
        if skip_test_at_end:
            self.cancelForTicket("DAOS-6427")
```